commons/src/main/java/io/ballerina/stdlib/graphql/commons/schema/SdlSchemaStringGenerator.java

```diff
@@ -16,43 +16,30 @@
  * under the License.
  */
 
-package io.ballerina.stdlib.graphql.compiler.schema.generator;
-
-import io.ballerina.projects.Project;
-import io.ballerina.projects.ProjectKind;
-import io.ballerina.stdlib.graphql.commons.types.DefaultDirective;
-import io.ballerina.stdlib.graphql.commons.types.Directive;
-import io.ballerina.stdlib.graphql.commons.types.DirectiveLocation;
-import io.ballerina.stdlib.graphql.commons.types.EnumValue;
-import io.ballerina.stdlib.graphql.commons.types.Field;
-import io.ballerina.stdlib.graphql.commons.types.InputValue;
-import io.ballerina.stdlib.graphql.commons.types.IntrospectionType;
-import io.ballerina.stdlib.graphql.commons.types.ScalarType;
-import io.ballerina.stdlib.graphql.commons.types.Schema;
-import io.ballerina.stdlib.graphql.commons.types.Type;
-import io.ballerina.stdlib.graphql.commons.types.TypeKind;
-
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
+package io.ballerina.stdlib.graphql.commons.schema;
```

Comment:
This should go into `graphql.commons.utils` package. We can move the existing `Utils` class there too.